test/dynamo/test_functions.py

```diff
@@ -3313,6 +3313,18 @@ def fn(x):
         ref = opt_fn(x)
         self.assertEqual(ref, res)
 
+    def test_frozenset_construction(self):
```

Comment:
Actually, can you please add a unit test that checks reconstruction of frozenset?

Reply:
added.